<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1361</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>QrcFiles/images/backgrounds/settings.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="player2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>543</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(QrcFiles/images/buttons/input.png);
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="player3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>543</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(QrcFiles/images/buttons/input.png);
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="player4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>543</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(QrcFiles/images/buttons/input.png);
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="player1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>543</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(QrcFiles/images/buttons/input.png);
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>20</y>
      <width>108</width>
      <height>107</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(QrcFiles/images/buttons/save.png);
  border: none;
  color: white;
  padding: 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="QrcFiles/resources1.qrc"/>
 </resources>
 <connections/>
</ui>
